### PR TITLE
Rycar/docupdate

### PR DESCRIPTION
**Pull request by ChefRycar**
Updated self guided workflow.

Pre-existing docs have been pushed to the "under-development" branch, as we won't be providing install instructions for the v1 trials.

### CODE REVIEW

File: simple_cookbook_workflow.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+6. Stage your changes and commit them:
+
+        git add .
+        git commit -m "this is the best change ever!"
+
+7. Run lint/unit/syntax tests locally first:
+
+        foodcritic .
+        rubocop .
+        rspec .
```
Comment (binamov) [inline simple_cookbook_workflow.md]: thanks for fixing those! we need to confirm rspec will run as is, sometimes it has to be `chef exec rspec`

File: simple_proxy_workflow.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+* In the Address box, type 127.0.0.1
+
+* In the Port box, type 1080
+
+* Click Apply
+
+You should now be able to connect to your Delivery Server, Chef Server, and Nodes via your browser.
+
+#### Continue to the [Local Development Cookbook workflow](simple_cookbook_workflow.md) doc, and let's stage a change for review!
```
Comment (binamov) [inline simple_proxy_workflow.md]: git's complaining here, wants a new line

File: simple_cookbook_workflow.md (hunk 1 above)
Comment (ChefRycar) [inline simple_cookbook_workflow.md]: I put the chfedk ruby in the path for the workstation node, so this shouldn't be an issue, but I'll confirm real quick.

Comment (ChefRycar) [inline simple_cookbook_workflow.md]: Confirmed!

File: simple_proxy_workflow.md (hunk 2 above)
Comment (ChefRycar) [inline simple_proxy_workflow.md]: Fixed.

**Comment by binamov** [comment]
:+1: 
:shipit:

**Pull request merged**